treeherder/seta/runnable_jobs.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+import requests
+from redo import retry
+
+HEADERS = {
+    'Accept': 'application/json',
+    'User-Agent': 'treeherder-seta',
+}
+LOG = logging.getLogger(__name__)
```

Comment:
I would prefer to change `logger` for `LOG`.
I try to use upper case variables at the beginning of a file to indicate that they're constant.
Otherwise, the variable `logger` would be only lowercase one among upper case variables.

Does this work for you?